src/test/e2e/e2e_helm_test.go

```diff
@@ -0,0 +1,31 @@
+package test
+
+import (
+	"fmt"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestE2eExampleHelm(t *testing.T) {
+	//defer e2e.cleanupAfterTest(t)
+
+	//run `zarf init`
+	output, err := e2e.execZarfCommand("init", "--confirm")
+	require.NoError(t, err, output)
+
+	path := fmt.Sprintf("../../../build/zarf-package-test-helm-releasename-%s.tar.zst", e2e.arch)
+
+	// Deploy the charts
+	output, err = e2e.execZarfCommand("package", "deploy", path, "--confirm")
+	require.NoError(t, err, output)
+
+	// Verify the configmap was properly templated
+	kubectlOut, _ := exec.Command("kubectl", "-A", "get", "pods", "--selector=app.kubernetes.io/name=etcd", "--no-headers").Output()
```

Comment:
Probably doesn't have to be this PR (but feel free if you want to 😆) but we should probably update our internal `GetPods()` [function](https://github.com/defenseunicorns/zarf/blob/fdaff86627fa1e2bba5bd5cf612aa6f1d50dce31/src/internal/k8s/pods.go#L80) to take in a `metav1.ListOptions{}` object so we could use that function instead of having to use kubectl on the host.

I think that would be useful for other tests / things in the future, I'll add an issue in the backlog for that.